app/ai/voice/agents/breeze_buddy/agent/turn_strategies/validators.py

```
@@ -0,0 +1,111 @@
+                config.stop_strategies is not None and len(config.stop_strategies) > 0
+            )
+
+            # It's okay to only specify stop strategies (start uses defaults)
+            # But if specifying start, should probably specify stop too
+            if has_start and not has_stop:
+                logger.warning(
+                    "Turn strategy config has start strategies but no stop strategies. "
+                    "Will use default transcription-based stop strategy."
+                )
+
+        return True, None
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)